recipes/configs/dev/qwen3B_async_grpo.yaml

```diff
@@ -0,0 +1,153 @@
+# Config for async multi-node GRPO in dev/async_grpo.py
+# using a Qwen-3B Base model (which is known to train quickly on this task).
+#
+# This config assumes that you've run the following command before launching
+# this run:
+#   tune download Qwen/Qwen2.5-3B --output-dir /tmp/Qwen2.5-3B --ignore-patterns "original/consolidated.00.pth"
+#
+# This config is meant to run on a whole node with 8 GPUs. You can tweak the number of workers here to change it
+# but note that some workers have currently been hardcoded in the prototype and we haven't yet tested on different
+# numbers of workers.
+
+# To launch, run the following:
+# tune run dev/async_grpo --config recipes/configs/dev/qwen3B_async_grpo.yaml
```

Comment:
Is this right? I thought the recipe name was `dev/async_grpo_full_finetune_distributed`

Reply:
Great catch thanks